docs/detections/api/rules/rules-api-create-single-rule-exception-item.asciidoc

```diff
@@ -0,0 +1,86 @@
+[[exceptions-api-create-rule-default-exception-item]]
+=== Create exceptions for a rule 
+
+Allows you to create exception items that are associated with a specified rule `id`.
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/detection_engine/rules/<id>/exceptions`
+
+//include a tip on how to find a rule's ID
```

Comment:
@yctercero how can users quickly find the ID of the rule they want to add an exception to?

Reply:
Great question. I'm not sure that we do provide a quick easy way. If they know the rules `rule_id`, they can fetch the rule and find it that way.

Reply:
Removed this comment and doc'd the request URL params in [585ed2c](https://github.com/elastic/security-docs/pull/4021/commits/585ed2cb75a83d00203852718b0a7d5fb610f4f6).